src/pages/get-started/api-security.md

```diff
@@ -7,6 +7,216 @@ This topic describes best practices for [API security](https://owasp.org/www-pro
+
+The following command will add a new connection to the Redis server:
+
+Redis server:
+- Host: 195.34.23.5
+- Port: 9345
+- Password: s0M3StR0NgP@SsW0Rd
+- User: SomeUser
+
+```bash
+$ bin/magento setup:config:set \
+    --backpressure-logger=redis \
+    --backpressure-logger-redis-server=195.34.23.5 \
+    --backpressure-logger-redis-port=9345 \
+    --backpressure-logger-redis-timeout=1 \
+    --backpressure-logger-redis-persistent=persistent \
+    --backpressure-logger-redis-db=3 \
+    --backpressure-logger-redis-password=s0M3StR0NgP@SsW0Rd\
+    --backpressure-logger-redis-user=SomeUser \
+    --backpressure-logger-id-prefix=some_pref
+```
+
+After the command is executed, the following configuration is added to the `app/etc/env.php` file.
+
+```php
+[
+//...
+    'backpressure' => [
+        'logger' => [
+            'type' => 'redis',
+            'options' => [
+                'server' => '195.34.23.5',
+                'port' => 9345,
+                'timeout' => 1,
+                'persistent' => 'persistent',
+                'db' => '3',
+                'password' => 's0meStr0ngPassw0rd',
+                'user' => 'SomeUser'
+            ],
+            'id-prefix' => 'some_pref'
+        ]
+    ]
+//...
+];
+```
+
+Use the following commands to enable rate limiting:
+
+1. Enable (`1`) or disable (`0`) rate limiting for placing orders:
+
+    ```bash
+    $ bin/magento config:set sales/backpressure/enabled 1
+    ```
+   
+1. Set the request limit per guest (IP address):
+
+    ```bash
+    $ bin/magento config:set sales/backpressure/guest_limit 100
+    ```
+   
+1. Set the request limit for authenticated customers:
+
+    ```bash
+    $ bin/magento config:set sales/backpressure/limit 10
+    ```
+   
+1. Set the period of time (in seconds) for the request limit. Supported values `60`, `3600`, `86400` seconds. This time period is multiplied by three to calculate the timeout period:
+
+    ```bash
+    $ bin/magento config:set sales/backpressure/period 60
+    ```
+   
+If you need to check a configuration, use the following CLI command:
+
+Example:
+
+```bash
+$ bin/magento config:show | grep backpressure
+```
+Response:
+
+```terminal
+sales/backpressure/limit - 10
+sales/backpressure/guest_limit - 100
+sales/backpressure/period - 3600
+sales/backpressure/enabled - 1
+```
+
+You can also enable and configure rate limiting in the user interface: `Stores` -> `Config` -> `Sales` -> `Sales` -> `Rate Limiting`:
+
+![Rate limiting section](../_images/api-security-rate-limiting.png)
+
+<InlineAlert variant="info" slots="text"/>
+
+If rate limiting was enabled for the payment information endpoint and the mutation functionality, but the connection to the service for store log request was not configured or configured incorrectly, then the restrictions will not be applied.
```

Comment:
I don't understand this note. You probably need a link to the place store log requests are configured.

Reply:
Done